### ISSUE TITLE
[Minor - Enhancement] Maybe good to consider unpin action on the Pinned Pane as well for the thought

### ISSUE THREAD

**Issue by harshitpthk** [n3xah/issues#18]
![image](https://github.com/user-attachments/assets/1f57f828-860d-4d26-bb9b-d04556b0fffd)

In case my usual list of thoughts becomes large unpin action might be hard to find in the list of thoughts so having an unpin action in the pinned pane per thought might be helpful for some users.

**Comment by veronicamarie24** [comment]
@harshitpthk I agree, thankfully this will be possible in our new homepage when our spaces feature is released :D
